### ISSUE TITLE
Visualize tags

### ISSUE THREAD

**Issue by MauroLima-hub** [minbrowser/min#2424]
When add new bookmarks, I would like to see the tags I already created, so I could click them, instead creating new ones.
At this moment, I created 2 or 3.
When I add a new bookmark, they don't pop!

**Comment by PalmerAL** [comment]
Yeah, this would make sense. Currently, we predict which tags are likely to apply to an item, and show those. We could also show the 2-3 most recently used ones.
